data/mainnet/tokens.js

```diff
@@ -8854,4 +8854,151 @@ module.exports = {
     rowTooltip: 'Out of Range',
     disableAutoSwap: true,
   },
+  aura_auraBAL: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: [
+      VAULT_CATEGORIES_IDS.AURA,
+      VAULT_CATEGORIES_IDS.BALANCER,
+      VAULT_CATEGORIES_IDS.ETH20,
```

Comment:
This category is for staked ETH2.0 wrappers, so this does not belong in the category

Reply:
Fixed!